ocaml/idl/datamodel_host.ml

```diff
@@ -2176,6 +2176,10 @@ let t =
              user should follow to make some updates, e.g. specific hardware \
              drivers or CPU features, fully effective, but the 'average user' \
              doesn't need to"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:String
+            ~default_value:(Some (VString "")) "last_update_hash"
+            "The SHA256 checksum of updateinfo of the most recently applied \
```

Comment:
An update that has been applied ma still not be active because it might require a restart of some kind. I assume we updating this when applying the update even when this may mean that it is not yet active.

Reply:
Yes. The "not yet active" is tracked by pending guidance. Overall, the "host.last_update_hash" + host pending guidance lists could reflect the status of the update.